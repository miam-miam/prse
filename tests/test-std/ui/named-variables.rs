use prse::parse;

fn main() {
    let l = "test: 5";

    parse!(l, "test: {var_that_definitely_exists}");
    parse!(l, "test: {var_that_definitely_exists:,:}");
    parse!(l, "test: {var_that_definitely_exists:,:5}");
    parse!(l, "test: {var_that_definitely_exists:,:0}");
}
