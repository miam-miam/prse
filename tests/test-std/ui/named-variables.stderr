error[E0425]: cannot find value `var_that_definitely_exists` in this scope
 --> ui/named-variables.rs:6:15
  |
6 |     parse!(l, "test: {var_that_definitely_exists}");
  |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `var_that_definitely_exists` in this scope
 --> ui/named-variables.rs:7:15
  |
7 |     parse!(l, "test: {var_that_definitely_exists:,:}");
  |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `var_that_definitely_exists` in this scope
 --> ui/named-variables.rs:8:15
  |
8 |     parse!(l, "test: {var_that_definitely_exists:,:5}");
  |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `var_that_definitely_exists` in this scope
 --> ui/named-variables.rs:9:15
  |
9 |     parse!(l, "test: {var_that_definitely_exists:,:0}");
  |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope
